frontend/src/store/wallet.ts

```diff
@@ -265,8 +265,11 @@ export default function useWalletStore() {
       }
 
       // Set Umbra and StealthKeyRegistry classes
-      umbra.value = new Umbra(provider.value, newChainId);
-      stealthKeyRegistry.value = new StealthKeyRegistry(signer.value);
+      // We have to use markRaw here to prevent Vue from making our ethers objects into proxies:
+      //   https://github.com/vuejs/core/issues/3024
+      //   https://github.com/dcgtc/dgrants/blob/2732f3107fd497b8c7f6ba7a13fcfb9f9f0e89e9/app/src/store/wallet.ts#L8-L20
```

Comment:
Nit, but let's just copy that good dgrants comment into the top of this file the same way (or tweak it however you see fit). But since that repo is dead, just want to make sure the info is preserved here in case that repo disappears one day

Reply:
Good idea, done in a58bf34